projects/playground/src/app/pages/grid/grid.models.ts

```diff
@@ -29,4 +29,5 @@ export interface IInputs {
     showHeaderRow: boolean;
     customFilter: boolean;
     useCardView: boolean;
+    showFirstLastButtons: boolean;
```

Comment:
```suggestion
    showFirstLastButtons?: boolean;
```
to preserve backcompat?